### PR TITLE
Added part b limitation

### PR DESCRIPTION
**Pull request by carolinexuxy**

### CODE REVIEW

File: report.tex [hunk 1]
```diff
@@ -142,7 +143,16 @@ \subsection{Model vs Baseline Models}
 Note, each row of this table represents the performance of the modifications stated in the "Model" column. Each modification is has been added onto the baseline model from Part A, Q3.\\
 \subsection{Model Performance}
 Because our model is made up of many small modifications, the final performance can be broken down into smaller parts. We can modifications like 
+
+
 \subsection{Model Limitation}
+One significant limitation of our proposed model lies in the hardware access. Since the only accessible machines to us are our laptops, the restriction in computation power arises naturally, making it hard for us to implement a network with more than two hidden layers. This constraint hinders our ability to develop deeper architectures and potentially deprives the model's capacity to represent data in scenarios where complicated patterns exist (such as language processing or image recognition).\\
```
Comment (skoooch) [inline report.tex]: This model wont ever really be in scenarios where we need to perform image recognition its just collaborative filtering.

Comment (skoooch) [reply report.tex]: just get rid of that last part of the sentence

**Review by skoooch — CHANGES_REQUESTED** [review]
all good just that one sentence in first paragraph

File: report.tex (hunk 1 above)
Comment (carolinexuxy) [reply report.tex]: Edit made. pls check

**Pull request merged**